CHANGELOG.md

```diff
@@ -1,218 +1,364 @@
 ## Changelog
 
+### v1.4.14
+
+#### Updated
+
+- Changed all instances of the Panel component to render contents only on click.
+
```

Comment:
Similar to what's on line 34, add that the Reusable Components package was updated to 1.3.19.

Reply:
Okay!